### ISSUE TITLE
CLI command to display the agents fingerprint(s)

### ISSUE THREAD

**Issue by catsby** [hashicorp/nomad#117]
For operating or debugging, it would be helpful if I could run a command on a node and get some information on the results of the Fingerprints, both which ones ran, and what they recorded. This would be helpful for understanding why a job isn't placing where I thought it would, or debugging a Fingerprint. 

Ex: 

```console
$ nomad node-status -fingerprint
Fingerprints ran:
- Java
  - version: 1.7...
- Docker
- CPU:
  - cores: 24
  - mhz: 9001
- Blah..
- Blah
```

**Comment by iverberk** [comment]
Is there still a need for the -fingerprint flag or is this fixed by #313?

**Comment by dadgar** [comment]
I think it is largely solved by that PR. I think it is best we close this PR. Thanks so much!
